### PR TITLE
Update export-or-backup-using-dumpling.md

### PR DESCRIPTION
**Pull request by yikeke**
### What is changed, added or deleted? (Required)

Align https://github.com/pingcap/docs-cn/pull/3541 that applies to other 4 release branches.

After this PR is merged, this file will be updated soon in https://github.com/pingcap/docs/pull/3453 on master and v4.0 branch. FYI @ran-huang @lichunzhu 

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3541/
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [x] Might cause conflicts

### CODE REVIEW

**Comment by yikeke** [comment]
Updated. PTAL again @ran-huang

File: export-or-backup-using-dumpling.md [hunk 1]
```diff
@@ -10,28 +10,38 @@ This document introduces how to use the [Dumpling](https://github.com/pingcap/du
 
 For backups of SST files (KV pairs) or backups of incremental data that are not sensitive to latency, refer to [BR](/br/backup-and-restore-tool.md). For real-time backups of incremental data, refer to [TiCDC](/ticdc/ticdc-overview.md).
 
+For detailed usage of Dumpling, use the `--help` command to view it.
```
Comment (ran-huang) [inline export-or-backup-using-dumpling.md:13]: Why not include the English user guide (https://github.com/pingcap/dumpling/blob/master/docs/en/user-guide.md)?

File: export-or-backup-using-dumpling.md [hunk 2]
```diff
@@ -41,42 +51,87 @@ For example, you can export all records that match `id < 100` in `test.sbtest1`
...
+  -o /tmp/test \
+  --filter "employees.*"
+  --filter "*.WorkOrder"
+```
+
+The above command exports all the tables in the `employees` database and the `WorkOrder` tables in all databases.
+
+#### Use the `-B` or `-T` command to filter data
+
+Dumpling can also export specific databases or tables with the `-B` or `-T` command.
```
Comment (ran-huang) [inline export-or-backup-using-dumpling.md:105]:
```suggestion
Dumpling can also export specific databases with the `-B` command or specific tables with the `-T` command.
```

File: export-or-backup-using-dumpling.md (hunk 1 above)
Comment (yikeke) [inline export-or-backup-using-dumpling.md:13]:
```suggestion
For detailed usage of Dumpling, use the `--help` command or refer to [Dumpling User Guide](https://github.com/pingcap/dumpling/blob/master/docs/en/user-guide.md).
```

Comment (yikeke) [reply export-or-backup-using-dumpling.md]: Nice catch. Updated.

**Pull request merged**